elephant/asset/asset.py

```diff
@@ -1854,6 +2141,14 @@ def cluster_matrix_entries(mask_matrix, max_distance, min_neighbors,
             stretching increases from 1 to `stretch` as the direction of the
             two elements moves from the 45 to the 135 degree direction.
             `stretch` must be greater than 1.
+        working_memory : int or None, optional
+            The sought maximum memory for temporary distance matrix chunks.
+            When None (default), no chunking is performed. This parameter
+            is passed directly to
+            ``sklearn.metrics.pairwise_distances_chunked`` function and it
+            has no influence on the outcome matrix. Instead, it control the
+            memory VS speed trade-off.
```

Comment:
The unit of this appears to be MiB according to https://github.com/scikit-learn/scikit-learn/blob/95119c13af77c76e150b753485c662b7c52a41a2/sklearn/utils/__init__.py#L919
Maybe you could add that here in the documentation.

Reply:
done